controllers/operatorpolicy_status.go

```diff
@@ -211,8 +211,10 @@ func calculateComplianceCondition(policy *policyv1beta1.OperatorPolicy) metav1.C
 	}
 
 	idx, cond = policy.Status.GetCondition(installPlanConditionType)
+
```

Comment:
Nit: can you remove these extra newlines? I don't think they are needed by the linter; the other similar stanzas don't have them.